contracts/interfaces/SettingsInterface.sol

```diff
@@ -210,6 +248,21 @@ interface SettingsInterface {
      */
     function unpauseLendingPool(address lendingPoolAddress) external;
 
+    /**
+        @notice Add a new Node Component with its version.
+        @param componentName name of the component
+        @param minVersion minimum component version supported
+     */
+    function addNewComponent(bytes32 componentName, uint256 minVersion) external;
+
+    /**
+        @notice Set a new version for a Node Component.
+        @param componentName name of the component
+        @param newMinVersion minimum component version supported
+     */
+    function updateComponentVersion(bytes32 componentName, uint256 newMinVersion)
+        external;
+
```

Comment:
I think the remove function is missing.

Reply:
I added it, thanks for catching this

Reply:
Correct, I added this function in the Interface too.